.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,19 @@
-### What was wrong?
+## What was wrong?
 
+Issue #
 
+## How was it fixed?
 
-### How was it fixed?
+Summary of approach.
 
+### To-Do
 
+[//]: # (Stay ahead of things, add list items here!)
+- [ ] Clean up commit history
+
+[//]: # (See: https://<RTD_NAME>.readthedocs.io/en/latest/contributing.html#pull-requests)
```

Comment:
This should replace `<RTD_NAME>` with `py-trie` but I don't think we have a "contributing" section so we can just remove this line?